monai/metrics/meandice.py

```diff
@@ -10,15 +10,17 @@
 # limitations under the License.
 
 import warnings
-from typing import Union
+from typing import Optional, Union
 
 import torch
 
 from monai.metrics.utils import do_metric_reduction, ignore_background
 from monai.utils import MetricReduction
 
+from .metric import Metric
 
-class DiceMetric:
+
+class DiceMetric(Metric):
     """
     Compute average Dice loss between two tensors. It can support both multi-classes and multi-labels tasks.
     Input `y_pred` (BNHW[D] where N is number of classes) is compared with ground truth `y` (BNHW[D]).
```

Comment:
same comments in `monai/metrics/hausdorff_distance.py`

Reply:
updated in the latest commit.
Thanks.